athina/steps/api.py

```diff
@@ -27,6 +27,7 @@ class ApiCall(Step):
     url: str
     method: str
     headers: Optional[Dict[str, str]] = None
+    params: Optional[Dict[str, str]] = None
```

Comment:
Add params in the Attributes of the class

Reply:
Done